project_app/components/Logout.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { SessionContext } from './SessionContext'; // Assuming SessionContext is imported from a separate file
+import { Stack, Alert } from '@mui/material'; // Assuming you're using Material-UI
+
+function LogoutComponent() {
+  const isLoggedOut = true;
+  const handleLogout = () => {};
+
+  return (
+    <SessionContext.Provider value={{ isLoggedOut, handleLogout }}>
```

Comment:
i believe the sessioncontext should be applied in _app.tsx so that every file will be accounted for